Core/src/com/serotonin/m2m2/db/dao/SystemSettingsDao.java

```diff
@@ -865,6 +871,18 @@ public void validate(Map<String, Object> settings, ProcessResult response, Permi
             }
         }
 
+        setting = settings.get(TIMEZONE);
+        if(setting != null) {
+            if(setting instanceof String) {
+                String zoneId = (String) setting;
+                TimeZone tz = TimeZone.getTimeZone(zoneId);
+                if(tz == null || !tz.getID().equals(zoneId))
```

Comment:
Does the retrieved TimeZone instance sometimes have a different ID? Under what circumstances? Please add some details and an example in a comment.